src/plugins/data/common/search/aggs/agg_type.ts

```diff
@@ -27,7 +27,8 @@ type PostFlightRequestFn<TAggConfig> = (
   searchSource: ISearchSource,
   inspectorRequestAdapter?: RequestAdapter,
   abortSignal?: AbortSignal,
-  searchSessionId?: string
+  searchSessionId?: string,
+  searchId?: string
```

Comment:
Could we give this a different name, it seems like `searchSessionId` and `searchId` are pretty similar. What about `inspectorId` to align with `inspectorRequestAdapter`?

Reply:
`inspectorId` is quite generic. Maybe something like `requestGroup` then.
I agree that `searchSessionId` and `searchId` are pretty close, but also what they represent is very similar.
No strong opinion here, I can change it, but probably wait for someone else feedback on this naming thing.